translation/run_translation_module.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+# --------------------------------------------------------
```

Comment:
Let's be consistent about the license header here (it's important because those headers are automatically parsed by our bot and it will raise warning / error if the format is not recognized)